### PR TITLE
Fix some small typos in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by brianhelba**
The most important issue was the incorrect GitHub group.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -44,10 +44,10 @@ is the checklist:
 * All tests pass
 * Consensus is reached. This usually means that at least one reviewer added a `LGTM` comment
 and a reasonable amount of time passed without anyone objecting. `LGTM` is a
-shortcut for _Looks Good to Me_.
+acronym for _Looks Good to Me_.
```
Comment (zachmullen) [inline CONTRIBUTING.md]: an*

Comment (jeffbaumes) [inline CONTRIBUTING.md]: "a acronym" -> "an acronym"

**Comment by brianhelba** [comment]
@jcfr Small addition to your https://github.com/girder/girder/pull/981

File: CONTRIBUTING.md (hunk 1 above)
Comment (jeffbaumes) [inline CONTRIBUTING.md]: Jinx @zachmullen

Comment (brianhelba) [inline CONTRIBUTING.md]: How ironic.

**Comment by zachmullen** [comment]
LGTM

**Comment by jcfr** [comment]
:8ball:

**Pull request merged**